### PR TITLE
Feat/remove trip

### LINKED ISSUE #23: LocalStorage
## Value Statement
As a user, I want to save my trips to the app in order to see them anytime I want, change them or remove them from the app.

## Acceptance Criteria
- [x] save trip to localStorage
- [x] edit trip in localStorage
- [x] remove trip from localStorage

## Tasks
- [x] create function to add trips to localStorage
- [x] create function to get a single trip from localStorage
- [x] create function to edit a single trip
- [x] create function to remove a single trip

## Size
large

### PR DESCRIPTION
**Pull request by yvlange**
Now you can remove your saved trips again.
Thanks for your reviews!

### CODE REVIEW

File: src/components/Trips.js [hunk 1]
```diff
@@ -12,6 +16,12 @@ function Trips() {
...
 
+  function handleRemoveTrip(trip) {
+    removeTripFromLocalStorage(trip);
+    const newTrips = getTripsFromLocalStorage();
+    setTrips(newTrips);
+  }
+
   function renderTrips() {
     return trips.map((trips, index) => {
       return (
```
Comment (namirsab) [inline src/components/Trips.js:29]: What you are returning in here could be a component as well, (`TripCard` or something like this)

File: src/services/tripsStorage.js [hunk 2]
```diff
@@ -33,3 +33,13 @@ export function editSingleTripFromLocalStorage(id, updatedTrip) {
...
   localStorage.setItem("tripData", JSON.stringify(updatedTrips));
 }
+
+export function removeTripFromLocalStorage(tripId) {
+  const myTrips = getTripsFromLocalStorage();
+
+  const newTrips = myTrips.filter((trip) => {
+    return trip.id !== tripId;
+  });
+  console.log(newTrips);
```
Comment (namirsab) [inline src/services/tripsStorage.js]: 🔥

File: src/components/Trips.js [hunk 3]
```diff
@@ -22,9 +32,17 @@ function Trips() {
               alt="destination"
```
Comment (namirsab) [inline src/components/Trips.js]: I realized you are relying on the size you pass to unsplash for the images size. 
You should set a size for your image using css

File: src/styles/Trips.css [hunk 4]
```diff
@@ -28,4 +21,14 @@ h3 {
...
   text-decoration: none;
+  letter-spacing: 2px;
+}
+.randomImages__info {
+  display: flex;
+  justify-content: space-around;
+  margin-bottom: 0;
+}
+
+.randomImages__delete {
```
Comment (namirsab) [inline src/styles/Trips.css]: I think the button is pretty small riight now:
image

File: src/components/Trips.js [hunk 5]
```diff
@@ -12,6 +16,12 @@ function Trips() {
     setTrips(trip);
   }, []);
 
+  function handleRemoveTrip(trip) {
+    removeTripFromLocalStorage(trip);
```
Comment (namirsab) [inline src/components/Trips.js]: I'd ask for confirmation before removing. What if the user taps accidentally on the trashcan?

**Review by namirsab — CHANGES_REQUESTED** [review]
Hey @yvlange !
lovely little PR,
I've asked you to do a few little changes. I hope it's helpful!

File: src/services/tripsStorage.js (hunk 2 above)
Comment (yvlange) [reply src/services/tripsStorage.js]: One day I'll catch them all :D

File: src/components/Trips.js (hunk 5 above)
Comment (yvlange) [reply src/components/Trips.js]: That's a good point. I'll add that

**Pull request merged**